packages/cli/src/commands/teams/switch.ts

```diff
@@ -9,15 +9,20 @@ import getTeams from '../../util/teams/get-teams';
 import listInput from '../../util/input/list';
 import { Team, GlobalConfig } from '@vercel-internals/types';
 import { writeToConfigFile } from '../../util/config/files';
+import { Output } from '../../util/output';
 
-const updateCurrentTeam = (config: GlobalConfig, team?: Team) => {
+const updateCurrentTeam = (
+  config: GlobalConfig,
+  output: Output,
+  team?: Team
+) => {
   if (team) {
     config.currentTeam = team.id;
   } else {
     delete config.currentTeam;
   }
 
-  writeToConfigFile(config);
+  writeToConfigFile(config, output);
```

Comment:
I won't block on it, but I would have preferred `output` as the first arg, kinda similar to how we have `client` as the first arg for a lot of other util functions.

Reply:
I agree with the general pattern @TooTallNate mentions. I also wouldn't block on it.